travis.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+extends =
+    https://raw.githubusercontent.com/collective/buildout.plonetest/master/travis-5.x.cfg
```

Comment:
seriously? I'll need to take a look at that package to fix it; I have not used a travis.cfg file in eons: that's not best practice anymore; IMO, best practice is to use the container infrastructure on Travis CI.